src/assets/source/styles/partials/_serlo_home.scss

```diff
@@ -286,6 +295,55 @@ $homeBreakpoint-lg-max: ($homeBreakpoint-lg-min - 1);
   }
 }
 
+.home-calls-wrapper {
+  background: #fff;
+  padding: 20px 0px;
+  margin-bottom: 100px;
+}
+
+.home-calls-header {
+  text-align: left;
+  padding: 0 0 20px 0;
+  margin-top: 20px;
+  color: #333;
+    h2 {
+      margin-bottom: 2px;
+      font-size: 30px;
+      font-weight: bold;
+  }
+}
+.home-calls-column {
+  font-size: 16px;
+  padding: 10px 0;
+  text-align: left;
+
+  .home-emph-link {
+    font-size: 25px;
+    margin-top:20px;
+    &:focus, &:hover, &:active {
+      color: $serloGreen;
+    }
+    i {
+      color: $serloGreen;
+    }
+  }
+}
+
+.calls-column-1 {
+  @media (max-width: 992px){
+    padding-bottom: 50px;
+  }
+}
+
+.home-row .extra-icon {
+  text-align: left;
+  padding: 10px 0px;
+  @media (min-width: 1199px) {
```

Comment:
You're right, the variables didn't seem to work w/o restarting vagrant so I did it with these, I should clean it up.